app/services/master_controller.py

```
@@ -1524,49 +1526,129 @@ async def get_system_status(self, user_id: str) -> Dict[str, Any]:
             
-            # Mock performance data (would be real in production)
-            performance_today = {
-                "trades": 5,
-                "profit_loss": 125.50,
-                "win_rate": 80.0,
-                "best_trade": 45.30,
-                "worst_trade": -12.10
-            }
+            # Get real portfolio performance history from database
+            try:
+                async with AsyncSessionLocal() as db:
```

Ruff (0.12.2)

Undefined name `AsyncSessionLocal`

(F821)